index.js

```diff
@@ -115,9 +115,11 @@ module.exports = {
 			}
 		],
 		'no-alert': 'error',
+		"no-autofix/prefer-const": "error",
 		'no-bitwise': 'warn',
-		'no-console': 'off',
+		'no-console': 'error',
```

Comment:
@dotpointer also replace with `no-autofix/no-console` and `no-autofix/no-debugger`

Reply:
Aha, OK.

Reply:
Replaced.